api/v1beta1/common.go

```diff
@@ -33,3 +38,69 @@ type extraConfigmapsMounts struct {
 	// Config map subpath for mounting, defaults to configmap root.
 	SubPath string `json:"subPath"`
 }
+
+type CommonOptions struct {
+	// +operator-sdk:csv:customresourcedefinitions:type=spec
+	// +kubebuilder:validation:Optional
+	// +kubebuilder:default="local-storage"
+	// StorageClass used to create any test-operator related PVCs.
+	StorageClass string `json:"storageClass"`
+
+	// +operator-sdk:csv:customresourcedefinitions:type=spec
+	// +kubebuilder:validation:Optional
+	// +kubebuilder:default:=""
+	// A URL of a container image that should be used by the test-operator for tests execution.
+	ContainerImage string `json:"containerImage"`
+
+	// +operator-sdk:csv:customresourcedefinitions:type=spec
+	// BackoffLimimt allows to define the maximum number of retried executions (defaults to 6).
```

Comment:
This mistake was already there but since there are changes that are required as part of this PR we can update this as well.

```suggestion
	// BackoffLimimt allows to define the maximum number of retried executions (defaults to 0).
```